src/Nethermind/Nethermind.Specs/ChainSpecStyle/ChainParameters.cs

```diff
@@ -120,6 +120,14 @@ public class ChainParameters
 
         public UInt256? TerminalTotalDifficulty { get; set; }
 
-        public long? Eip1153Transition { get; set; }
+        /// <summary>
+        /// this feild will indicate the timestamp at which this EIP1153 will be enabled.
+        /// </summary>
+        public ulong? Eip1153TransitionTimestamp { get; set; }
```

Comment:
What's about the rest of EIPs?

Reply:
Should be added in their own PR's?

Reply:
Will be added with their PRs or by merging shandong